detection_rules/etc/version.lock.json

```diff
@@ -338,7 +338,7 @@
     "rule_name": "Google Drive Ownership Transferred via Google Workspace",
     "sha256": "98600fe4b1c0c882bb99021122279f31ce5cdd2266abf34b56bab33f0cb7f190",
     "type": "query",
-    "version": 3
+    "version": 5
```

Comment:
All Google Workspace rule versions are being bumped without receiving any changes.

Reply:
Confirmed rules are double bumping between branches 8.3 and 8.4. 
<img width="1216" alt="Screen Shot 2023-01-12 at 5 49 48 PM" src="https://user-images.githubusercontent.com/99630311/212198396-0bea84f6-505b-4374-bb22-e36878494cf9.png">

Reply:
Google Drive Ownership Transferred via Google Workspace - 07b5f85a-240f-11ed-b3d9-f661ea17fbce


Existing 8.3
8.3 -> Version 3 -> 98600fe4b1c0c882bb99021122279f31ce5cdd2266abf34b56bab33f0cb7f190

8.3 branch - update version lock
8.3 -> Version 4 -> 1c82ea9b65fada4ec684045bd8b3e5eaa0730b35b41ddef3dd151ff26a9d6be9

8.4 branch - update version lock
8.3 -> Version 5 -> 98600fe4b1c0c882bb99021122279f31ce5cdd2266abf34b56bab33f0cb7f190

It almost appears that in 8.3, the rule has an update, but then in 8.4, the change is reverted and the outcome is the same rule prior to the 8.3 update. Between these two branches the rule is identical, suggesting the change may be related to dynamically generated fields.

Reply:
The logic to this is in steps below:

8.3 branch
1. Latest major Google Workspace integration version is 2
2. Starting with the least minor version (2.0.0), check policy templates and Kibana compatible version which is 8.4
3. Since 8.3 < 8.4, go to the next major 1
4. Starting at 1.1.2 as the least minor, check all minors for major 1 until a compatible Kibana version matches
5. At 1.2.0 Kibana version is `^7.16 || ^8.0.0`, therefore it is compatible and assign 1.2.0 as recommended version

8.4 branch
1. Latest major Google Workspace integration version is 2
2. Starting with the least minor version (2.0.0), check policy templates and Kibana compatible version which is 8.4
3. Since 8.4 <= 8.4, it is compatible and assign 2.0.0 as the recommended version

Reply:
If you look at the google_workspace [PR](https://github.com/elastic/integrations/pull/3430/files#diff-6154ae00eee62ae6a565d22dd6300683900401391fd56207ab3566caca098ed4) they bumped to 8.4 for a breaking change feature, so I think it does make sense to min_stack our rules to align with the integration long term (IIF we are using the breaking change feature).